.github/send_alerts_to_slack.py

```diff
@@ -0,0 +1,95 @@
+"""
+
+Summarize the Frictionless Data JSON error output and if there are any errors, 
+send a summary message to a given Slack channel. Send a more detailed 
+error summary to std out and create a detailed txt file that can be accessed via GitHub
+artifacts.
+
+To run elsewhere, define SLACK_WEBHOOK in your repo's secrets and change the 
+top-level constants defined below. 
+
+`validate-csv.yml` can also be copied verbatim, assuming that the schema is 
+named `datapackage.json`.
+
+Links to...
+- Generate a Slack token
+    https://slack.com/apps/A0F7XDUAZ-incoming-webhooks
+- API for POST requests / formatting text:
+    https://api.slack.com/messaging
+
+
+"""
+
+import json
+import requests
+import os
+
+ERROR_FILE = 'error_summary.json'
```

Comment:
Should this be set in an environment variable or other config? Seems like that would make it easier to ensure the two actions are working with the same file.